XCTest.xcodeproj/project.pbxproj

```diff
@@ -255,6 +299,24 @@
 /* End PBXLegacyTarget section */
 
 /* Begin PBXNativeTarget section */
+		19D66F931DE6F57E00C112C0 /* SwiftXCTest-iOS */ = {
```

Comment:
Rather than adding a whole new target, it should be pretty straightforward to accomplish the same goal by adding `iphoneos` and `iphonesimulator` to the `SUPPORTED_PLATFORMS` build setting of the existing target. You'll probably need to also bring over a couple of additional settings like the iOS deployment target, but I would it expect it to mostly just work for a simple target like this.